### ISSUE TITLE
Cannot extract phone number from image

### ISSUE THREAD

**Issue by yns01** [tdurieux/leboncoin-api#8]
Hello, first thank you form you api.

There is an issue while parsing the GIF phone number. It raises an error : 
Invalid GIF 87a/89a header. I believe it is coming from this line :+1: 

` pixelGif.parse(phoneNumberAddress).then(function(images){`

Thanks.

**Comment by tdurieux** [comment]
Hello,

Can you give me the link to the announce that raise this error?

By the way, I do not recommend the use of the method getPhoneNumber because leboncoin limits drastically its usage.

**Comment by yns01** [comment]
Here one that does not work http://www.leboncoin.fr/locations/903589901.htm?ca=21_s
I saw that you use pixelGif. PixelGif in turns depends on these packages : 
```
PixelUtil= (require 'pixel-util').PixelUtil
GifReader= (require 'omggif').GifReader
```
I believe the error comes from one of those dependencies...

Oh ok, how should I do to get the phone number then ?

**Comment by tdurieux** [comment]
Hi,

I fix the issue with this commit 9b9e6641996ed14476e6ffafa4719e7f6ba98535.

> Oh ok, how should I do to get the phone number then ?
You can use it but you have to know that the quota is quite small (How many request? I don't know).
